index.html

```diff
@@ -5,43 +5,187 @@
+   <header>
+    <!-- Top Section -->
+    <div class="top-section">
+      <ul class="top-section-ul">
+        <li class="top-section-list"><img src="Assets\icons8-facebook-18.png" alt="facebook"/></li>
+        <li class="top-section-list"><img src="Assets\icons8-twitter-18.png" alt="twitter"/></li>
+        <li class="top-section-list">English</li>
+        <li class="top-section-list">My Page</li>
+        <li class="top-section-list">Logout</li>
+      </ul>
+    </div>
+
+    <!-- Navbar -->
+    <!--Mobile-->
+    <nav class="navbar-mobile">
+      <div class="menu">
+        <a> <img src="Assets\mobile_menu 1.png" class="menubar" alt="navbar"> </a>
+         </div>
+    </nav>
+
+    <nav class="mobile-menu">
+      <ul class="mobile-menu-list">
+       <img src="Assets\icons8-close-24 (6).png" class="closebtn" alt="close">
+       <li class="menu-list"> <a href="about.html" class="menu-link">About</a></li>
+     <li class="menu-list"> <a href="#about" class="menu-link">Program</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Join</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Sponsor</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">News</a></li>
+    </ul>
+     </nav>
+    
+      <!-- Nav Items -->
+      <nav class="navbar-container">
+        <a href="index.html" class="p-logo"><p class="logo">WIT</p></a>
+        <ul class="navbar-items">
+          <li>
+            <a href="about.html" class="menu-link-desktop">About</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Program</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Join</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Sponsor</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">News</a>
+          </li>
+
+          <li class="list-items">
+            <a href="#">WITMP 2023</a>
+          </li>
         </ul>
-      </div>
+      </nav>
+  </header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="theme-info">
+  <!-- Hero -->
+  <div class="first-section">
+  <div>
+  <h4 class="theme-intro">"Great News!"</h4>
+  <h2 class="theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h2>
+</div>
+
+<div class="intro">
+<h4 class="intro-text">
+Are you a woman trying to navigate the tech space? then this programme is for you. join us in Rivers State for the Women In Tech mentorship program 2023 edition.
+</h4>
+</div>
 
-      <!-- Navbar -->
-      <div class="navbar">
-        <!-- Logo -->
+<div class="venue">
+<h3 class="date-text">
+6th - 7th JULY 2023
+</h3>
+<h5 class="location">
+@Landmark Hotel, Rivers State.
+</h5>
+</div>
+</div>
 
-        <!-- Nav Items -->
+<div class="second-section">
+  <div class="program">
+    <h4 class="main-program">Main Program</h4>
+    <hr class="one">
```

Comment:
- [ ] please add margin left and right to the main program cards. The bottom and top as well
 - [ ] you are missing the underlined text (see the whole program) under the cards. Please add that to align with the design. 
 - [ ] Kindly add margin left and right to the speaker section 
 - [ ] please make the lines under the headings shorter, like in the original design. Remember taking small details into consideration is a good developer skill. 
 - [ ] please kindly add the small gray line under each speaker title.
Your design

![Screenshot (1166)](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/18d9dcf3-98aa-425a-9530-ddd58e978910)
original
 
![Screenshot (1167)_LI](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/692a9278-fdfd-4f18-831e-b7f8bdf84359)

Reply:
Thank you so much for your review, I have fixed the padding and margin around the cards.
![speakers](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/111312788/2cd706b7-a533-446d-8ff9-a6f3bbf5e3f0)